src/modules/message.js

```diff
@@ -0,0 +1,66 @@
+const deepFreeze = require('deep-freeze');
+/**
+  @class Message
+  Convert the given parameters into a frozen (@see Object.deepFreeze()) message package
+  @param logLevel {string} - a message log level (@see ChainLink @class for additional info)
+  @param message {Object|Error} - a message payload. Either a text string or an Error object
+  @param metas {Object} - metadata
+**/
+class Message {
+  constructor(logLevel, message, ...metas) {
+    // if plain text
+    this.payload = {
+      level: logLevel || 'info',
+      text: message || '',
+      meta: {
+        instanceId: process.env.HOSTNAME,
+        notify: true
+      }
+    };
+
+    // if error
+    if (message instanceof Error) {
+      this.payload.text = message.message || 'Error: ';
+      this.payload.meta.stack = message.stack;
```

Comment:
people can have custom errors with properties they would like to log. Any chance we can include the entire error?